iOS/Sample/Sample.xcodeproj/project.pbxproj

```diff
@@ -468,41 +427,6 @@
 					"\"$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)\"",
 					"\"$(TOOLCHAIN_DIR)/usr/lib/swift-5.0/$(PLATFORM_NAME)\"",
 				);
-				OTHER_CFLAGS = (
-					"$(inherited)",
-					"-fmodule-map-file=\"${PODS_ROOT}/Headers/Public/FlipperKit/FlipperKit.modulemap\"",
-					"-fmodule-map-file=\"${PODS_ROOT}/Headers/Public/yoga/Yoga.modulemap\"",
-					"-fmodule-map-file=\"${PODS_CONFIGURATION_BUILD_DIR}/YogaKit/YogaKit.modulemap\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/CocoaAsyncSocket\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/CocoaLibEvent\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/ComponentKit\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/DoubleConversion\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/Flipper\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/FlipperKit\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/Folly\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/OpenSSL-Static\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/PeerTalk\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/RSocket\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/Yoga\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/YogaKit\"",
-					"-isystem",
-					"\"${PODS_ROOT}/Headers/Public/glog\"",
-					"-DFB_SONARKIT_ENABLED=1",
```

Comment:
No longer needed to specify this explicitly.

(All the other build settings never needed to be set explicitly.)